CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.23.1] - 2024-11-07
+
 ## [4.23.0] - 2024-10-30
```

Comment:
@ljakimczuk do you mind adding a changelog entry?

Reply:
It's in conflict though :) And with the changes I'd probably makes this a minor :)